app/controllers/AccountHomeController.scala

```diff
@@ -75,9 +80,18 @@ class AccountHomeController @Inject() (
   def startApplication: Action[AnyContent] =
     identify.async {
       implicit request =>
-        for {
-          draftId <- counterRepository.nextId(CounterId.DraftId)
-          _       <- userAnswersService.set(UserAnswers(request.userId, DraftId(draftId)))
-        } yield Redirect(navigator.startApplicationRouting(request.affinityGroup, DraftId(draftId)))
+        AuthUserType(request) match {
+          case None           =>
+            // TODO: Update error handling
+            Future.successful(Redirect(controllers.routes.JourneyRecoveryController.onPageLoad()))
+          case Some(authType) =>
+            for {
+              nextId      <- counterRepository.nextId(CounterId.DraftId)
+              draftId      = DraftId(nextId)
+              userAnswers  = UserAnswers(request.userId, draftId, lastUpdated = Instant.now(clock))
+              userAnswers <- userAnswers.setFuture(ApplicantUserType, authType)
+              _           <- userAnswersService.set(userAnswers)
+            } yield Redirect(navigator.startApplicationRouting(request.affinityGroup, draftId))
```

Comment:
The only small point would be that this redirect might naturally sit in the navigator, thoughts?